app/assets/stylesheets/themes/default/footer.scss

```diff
@@ -1,68 +1,132 @@
 footer {
-  display: grid;
-  width: 100%;
-  border-top: 1px solid $divider-light;
-  grid-template-columns: 1fr 1fr;
-  padding: 20px 30px 35px;
-
-  .footer-event-details {
-    border-bottom: 1px solid $divider-light;
-    padding-bottom: 20px;
+  color: white;
+  .overlay {
+    background-color: black !important;
+    opacity: 0.9 !important;
+    padding: 20px 30px 35px;
+  }
```

Comment:
Were these colors not sticking without the '!important'?

Reply:
Oh, I think I can remove those. Thanks for pointing that out.